src/components/ArticleCard.vue

```diff
@@ -1,231 +1,169 @@
 <template>
-  <b-col class="d-flex justify-content-center">
-    <b-card
-      no-body
-      :class="{'card--wide': full}"
-      class="card my-3 justify-content-center"
-      footer-tag="footer"
-      footer-bg-variant="white"
-      footer-border-variant="white"
-      footer-class="p-0 article-footer"
-    >
-      <button v-b-tooltip.hover.bottom.v-secondary="bookmarkTooltip" class="subscribe-btn" @click="handleClickBookmark()">
-        <icon v-if="!isSubscribed" icon="save" />
-        <icon v-else icon="saved" />
-      </button>
-      <b-card-body class="d-flex flex-column p-0">
-        <b-link :to="`/article/${articleId}`">
-          <div class="d-flex justify-content-between">
-            <div class="article-category">
-              {{ getCategory(category) }}
-            </div>
-          </div>
-          <div>
-            <b-card-title class="heading">
-              <h4>
-                {{ getTitle(title) }}
-              </h4>
-            </b-card-title>
-          </div>
-
-          <b-card-text class="article-excerpt" :class="{'article-excerpt--wide': full}">
-            {{ getArticleContent(blocks, excerptCount) }}
-          </b-card-text>
-        </b-link>
-      </b-card-body>
-      <template #footer>
-        <div class="d-flex justify-content-between">
-          <div>
-            {{ getDateTime(lastUpdatedAt) }}
-          </div>
-          <div>觀看數：{{ viewsCount }} | 編輯數：{{ editedCount }}</div>
-        </div>
-      </template>
-    </b-card>
-  </b-col>
+  <div class="article-card" :class="{'article-card__lg':lg}" @click="GotoArticle">
+    <div class="article-card__image">
+      <b-img-lazy :src="imgSrc" fluid-grow />
```

Comment:
![image](https://user-images.githubusercontent.com/39425103/197149016-31674d32-560e-4865-a48e-ebab6d500edf.png)
The left section needs to be implemented too.
The title of each block should be displayed.